rcl/test/CMakeLists.txt

```diff
@@ -63,6 +63,17 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation}
   )
 
+  rcl_add_custom_gtest(test_get_node_names${target_suffix}
+    SRCS rcl/test_get_node_names.cpp
+    ENV
+      ${extra_test_env}
+      EMPTY_TEST=
+      NORMAL_TEST=foo
```

Comment:
looks like a copy-paste artifact and should be removed. these are used to [test getenv in common .c](https://github.com/ros2/rcl/blob/d3a27df6e2dda9638acf2d0e57eefeb22197742f/rcl/test/rcl/test_common.cpp#L29-L37) and are not used for the test_get_node_names test